uint8 control_mode       # 0: 手动模式; 1: 自动模式; 2: 远控 (澳汀没有专门的远控模式)
uint8 battery_percent    # 电量百分比 [0-100]
bool is_charge
